otter/log/formatters.py

```diff
@@ -30,6 +30,61 @@
 NON_PEP3101_SYSTEMS = ('kazoo',)
 
 
+_fanout = None
+
+
+def add_to_fanout(observer):
+    """
+    :return: the global instance of :class:`FanoutObserver`
```

Comment:
I'm kinda weirded out that this returns the observer -- especially reading the use of it in setup.py. The thing that makes it weird is that if you look at the way it's used in setup.py and then try to use it in a similar way elsewhere you'll end up putting the same observer in multiple places which could be very weird. I think to emphasize that it's mutating a global observer I'd prefer to just have this return None and then require the user to explicitly call `get_fanout` when they want to add the observer.

However this is a minor nitpick so it's not really important.